bin/upload-dist-versioned.sh

```diff
@@ -17,10 +17,12 @@ require AWS_SECRET_ACCESS_KEY "${AWS_SECRET_ACCESS_KEY}"
 require AWS_REGION "${AWS_REGION}"
 require S3_BUCKET "${S3_BUCKET}"
 require VERSION_TAG "${VERSION_TAG}"
+require STAGING_RELEASE "${STAGING_RELEASE}"
```

Comment:
should you move this into the copy_staging_release_to_dist function? is it always required?

Reply:
good point. It's only required for that particular function. I'll fix this.